src/test/migrations/steps/20241128113639-step-delete-fra-2025-dashboard.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseProtocol } from 'server/db'
+
+// delete FRA 2025 dashboard
+export default async (client: BaseProtocol) => {
+  await client.query(`delete from assessment_fra_2025.node_ext where (props->>'region')::boolean = true;`)
```

Comment:
We should persist this somewhere, maybe in this issue or pull request to avoid recreating the partial object

Reply:
What are you referring to with `country`?

Sure, we can do that also